documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -240,54 +240,54 @@
-      <input name="transmission_dispersion" type="float" value="0" />
-      <input name="transmission_extra_roughness" type="float" value="0" />
-      <input name="subsurface" type="float" value="0" />
-      <input name="subsurface_color" type="color3" value="1, 1, 1" />
-      <input name="subsurface_radius" type="color3" value="1, 1, 1" />
-      <input name="subsurface_scale" type="float" value="1" />
-      <input name="thin_walled" type="boolean" value="false" />
-      <input name="normal" type="vector3">
+      <input name="base" type="float" value="0.8" uiname="Base" uifolder="Base"/>
+      <input name="base_color" type="color3" value="1, 1, 1" uiname="Base Color"  uifolder="Base"/>
+      <input name="diffuse_roughness" type="float" value="0" uiname="Diffuse Roughness"  uifolder="Diffuse"/>
+      <input name="specular" type="float" value="1" uiname="Specular" uifolder="Specular"/>
+      <input name="specular_color" type="color3" value="1, 1, 1" uiname="Specular Color" uifolder="Specular"/>
+      <input name="specular_roughness" type="float" value="0.1" uiname="Specular Roughness" uifolder="Specular"/>
+      <input name="specular_IOR" type="float" value="1.52" uiname="Index of Refraction" uifolder="Specular"/>
+      <input name="specular_anisotropy" type="float" value="0" uiname="Specular Aniostropy" uifolder="Specular"/>
+      <input name="specular_rotation" type="float" value="0" uiname="Specular Rotation" uifolder="Specular"/>
+      <input name="metalness" type="float" value="0" uiname="Metalness" uifolder="Specular"/>
+      <input name="transmission" type="float" value="0" uiname="Transmission" uifolder="Transmission"/>
+      <input name="transmission_color" type="color3" value="1, 1, 1" uiname="Transmission Color" uifolder="Transmission"/>
+      <input name="transmission_depth" type="float" value="0" uiname="Transmission Depth" uifolder="Transmission"/>
+      <input name="transmission_scatter" type="color3" value="0, 0, 0" uiname="Tranmission Scatter" uifolder="Transmission"/>
+      <input name="transmission_scatter_anisotropy" type="float" value="0" uiname="Tranmission Anisotropy" uifolder="Transmission"/>
+      <input name="transmission_dispersion" type="float" value="0" uiname="Tranmission Dispersion" uifolder="Transmission"/>
+      <input name="transmission_extra_roughness" type="float" value="0" uiname="Transmission Roughness" uifolder="Transmission"/>
+      <input name="subsurface" type="float" value="0" uiname="Subsurface" uifolder="Subsurface"/>
+      <input name="subsurface_color" type="color3" value="1, 1, 1" uiname="Subsurface Color" uifolder="Subsurface"/>
+      <input name="subsurface_radius" type="color3" value="1, 1, 1" uiname="Subsurface Radius" uifolder="Subsurface"/>
+      <input name="subsurface_scale" type="float" value="1" uiname="Subsurface Scale" uifolder="Subsurface"/>
+      <input name="thin_walled" type="boolean" value="false" uiname="Thin walled"/>
```

Comment:
Typo: walled -> Walled

Reply:
thin_walled, normal, tangent, and opacity can be grouped into a "Geometry" uifolder (to follow the UI in Maya)